src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/DynamicMethod.CoreCLR.cs

```diff
@@ -172,7 +172,9 @@ Signature LazyCreateSignature()
                     Span<ParameterCopyBackAction> shouldCopyBackParameters = new(ref argStorage._copyBack0, argCount);
 
                     StackAllocatedByRefs byrefStorage = default;
+#pragma warning disable 8500
                     IntPtr* pByRefStorage = (IntPtr*)&byrefStorage;
```

Comment:
@jkotas is there a way to update these method without suppression?

Reply:
No. The warning suppression is correct. It is the only way to make this build with latest Roslyn.